src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1039,7 +1039,7 @@ export default class DefaultAudioVideoController
     // Similar to `updateRemoteVideosFromLastVideosToReceive`, we use `subscribeFrameSent` to cache the previous
```

Comment:
nit: forget to update comment? Same as above

Reply:
Good nit :) will update these